extensions/common/context.h

```diff
@@ -108,7 +108,7 @@ struct RequestInfo {
   uint32_t destination_port = 0;
 
   // Source port of the client.
-  uint32_t source_port = 0;
+  uint64_t source_port = 0;
```

Comment:
```suggestion
  uint32_t source_port = 0;
```

Reply:
This is just to keep it consistent with destination_port

Reply:
No this is needed as we get back uint64 from ABI and if we cast to uint32.. it comes up as 0..

Reply:
Not sure why it comes up as 0, but if that is the case, could you also change destination_port to use uint64_t? If it is going to cause problem to source_port, it would cause the same problem to destination port, wouldn't it?

Reply:
It doesn't cause problem for destination port because when we get it, we get it as uint64.. https://github.com/istio/proxy/blob/master/extensions/common/context.cc#L151:12 and then cast it as uint32.. thus I had changed it to uint64 too.. but hadn't tested it so changed it back.. 
we should make it consistent and will change to uint64 in a different pr..